docs/content/en/cookbook-preparation.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Preparation
+description: 'Preparation demo'
+position: 11
+category: Cookbook
+---
+
+The **demo** will show the creation of a simple repository based on **NuxtJS**,
+which will be transformed into a Micro Service architecture using **VueMS**.
+
+<alert type="info">
+  A full installation of the respirator you can see in <a href="https://github.com/ergonode/vuems-demo" target="_blank">this place</a>.
+</alert>
+
+
+## Create clean NuxtJS project
+
+To build a **demo** we use the ready [CLI NuxtJS][nuxt-cli] to build applications.
```

Comment:
we use a ready [CLI NuxtJS] [nuxt-cli] **file / component / method ??** to build applications.

There's something missing in that sentence that specifies what this CLI is.